docs/modules/record-encryption/proc-configuring-record-encryption-filter.adoc

```diff
@@ -0,0 +1,49 @@
+// file included in the following:
+//
+// assembly-record-encryption.adoc
+
+[id='proc-configuring-record-encryption-filter-{context}']
+= Setting up the Record Encryption filter
+
+[role="_abstract"]
+This procedure describes how to set up the Record Encryption filter.
+Provide the filter configuration and the Key Encryption Key (KEK) selector to use. 
+The KEK selector maps topic name to key names.  
+The filter looks up the resulting key name in the KMS.
+
+.Prerequisites
+
+* An instance of Kroxylicious. 
+For information on deploying Kroxylicious, see the link:{github}[samples and examples^].
+* A config map for Kroxylicious that includes the configuration for creating virtual clusters and filters. 
+* A KMS is installed and set up for the filter with KEKs to encrypt records set up for topics.
+
+.Procedure
+
+. Configure a `RecordEncryption` type filter.
++
+.Example Record Encryption filter configuration
```

Comment:
Side note, I wonder if we should include a complete configuration example somewhere. We currently have the template and then examples for each KMS. Maybe it would be useful somewhere to glue them together, maybe in an Example Configurations section? Somewhere the user can directly copy a whole config and get something that nearly works if they change the KMS address.

(not for this refactor PR)